### PR TITLE
Adapt the temporary layout to Portus' interface

### PR DESCRIPTION
**Pull request by eotchi**
Just a couple of changes. 
It will give the following results:

adapted_layout

### CODE REVIEW

**Comment by flavio** [comment]
I think it's fine, we should add some spacing between the logo and the 3 buttons (View on github, download as zip,...)

**Comment by mssola** [comment]
+1 with what @flavio says. Plus, you've removed the tagline "Claim control of your Docker images!". I liked that :P

**Comment by eotchi** [comment]
Something like this?
with_space

**Comment by flavio** [comment]
Yes, that's definitely better. Can you add back the "Claim control of your Docker images!" text?

File: stylesheets/stylesheet.css [hunk 1]
```diff
@@ -111,8 +111,8 @@ a {
 .main-content h1, .main-content h2, .main-content h3, .main-content h4, .main-content h5, .main-content h6 {
   margin-top: 2rem;
   margin-bottom: 1rem;
-  font-weight: normal;
-  color: #159957; }
```
Comment (mssola) [inline stylesheets/stylesheet.css]: why? I actually like this.

**Comment by eotchi** [comment]
Yep. At this point, I'd suggest this:
font

**Comment by flavio** [comment]
:+1:

**Comment by mssola** [comment]
:+1:

File: stylesheets/stylesheet.css (hunk 1 above)
Comment (eotchi) [inline stylesheets/stylesheet.css]: That's because there is no green in Portus. 
#383e41 is a "gimp colour pick" of the darker blue used in the logo

**Pull request merged**